### DISCUSSION TITLE
Is it possible to mirror a ceph cluster between two seperate k8s clusters?

### DISCUSSION THREAD

**Discussion by d3xma** [rook/rook#11488] [Q&A] [answered]
Hi everybody,

is it possible to create RBD/Filesystem mirroring between two ceph clusters in seperate k8s clusters?
I am aware of the capability to do so in the same k8s cluster, but i want to achive the same with a secondary k8s cluster. 

Thanks and BR

**Comment by travisn** [comment]
Here are a couple topics on the mirroring between clusters:
- RBD mirroring: https://rook.io/docs/rook/latest/Storage-Configuration/Block-Storage-RBD/rbd-mirroring/
- CephFS mirroring: https://rook.io/docs/rook/latest/Storage-Configuration/Shared-Filesystem-CephFS/filesystem-mirroring/

**Reply by d3xma** [reply]
@travisn thanks for your reply. 
I already worked through all of those docs and the ceph ones as well, but the only reference I can find in all of them is: 

> “All ceph nodes on the primary cluster need to be able to talk to all nodes in the secondary.” 

Imo this is not enough information to set this up correctly. 
Which ports and services need to be accessible from both clusters? 
Is the remote cluster ip embedded in the bootstrap token?
How can I verify connectivity between the clusters? 

I guess this is somehow related to the external cluster setup, where the Cluster CRD must be enabled with host networking.

**Comment by d3xma** [comment, marked as answer]
I have found an answer to my question, also with the help of @travisn.

Yes, it is definitely possible.
To configure it properly it is crucial to enable the [host networking](https://rook.io/docs/rook/latest/CRDs/Cluster/ceph-cluster-crd/#host-networking) capability in the Cluster CRD. 
After that the cluster is accessible from outside the k8s network and can be reached by the remote ceph mons.
